bixlabs/authentication
# [AM-35] Dokerize app

## Description 
In order to use terraform we need to dokerize the api

## Stories
https://bixlabs.atlassian.net/browse/AM-35

## List of changes 
* add: docker
* add: docker-compose for dev
* add: docker-compose for prod
* add: fetch air dependency in the setup process

## Steps to Test or Reproduce
```bash
make docker-build-run # for prod
make docker-run-dev # for dev with live reload
```
When you see that the API is ready, go to
http://localhost:9000/swagger/index.html

## TODO

- [x] add documentation to README.md
